### PR TITLE
add a new feature for sharing Geopackage files

### PR DESCRIPTION
**Pull request by sumandari**
This PR refers to #118. It includes:
- Basic CRUD
- Backend validation for max filesize
- Notification email for moderation system
- Search function that shows the result in list view page.
plugins_geopackages_0

### CODE REVIEW

**Comment by dimasciput** [comment]
@sumandari can you add more tests please?

**Comment by timlinux** [comment]
@sumandari thanks for this - I think I wont ask you to change the menu etc for now, let's go with what you have. Can you deploy it as soon as possible so I can test it? Usual things apply about making sure the platform doesnt break.

File: qgis-app/geopackages/models.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+
+class GeopackageApprovedManager(models.Manager):
+    """Custom Queryset Manager for Unapproved GeoPackage"""
+    def get_queryset(self):
+        return super().get_queryset().filter(approved=True) \
+            .order_by('upload_date')
+
+
+class GeopackageRequireactionManager(models.Manager):
```
[the remark this answers is not in the document]
Comment (sumandari) [reply qgis-app/geopackages/models.py]: the class has been renamed

**Review by dimasciput — APPROVED** [review]

**Comment by timlinux** [comment]
When the user does something wrong, be sure to handle exceptions (or write your code so that we don't get exceptions is even better) and then feed back to the user what they did wrong.

Also please support .zip compressed package files too.

**Comment by timlinux** [comment]
Also @sumandari can you please make sure that markdown is supported for the description field?

**Pull request merged**